core/eval_test.go

```diff
@@ -0,0 +1,50 @@
+package core
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/dicedb/dice/testutils"
+	"gotest.tools/v3/assert"
+)
+
+func TestEvalSET(t *testing.T) {
```

Comment:
These tests have been recently updated. Please consolidate your tests against [https://github.com/DiceDB/dice/blob/f4b56e52b0ba5a13ac43ffe2f1e8bc259813dbb7/core/eval_test.go#L25](this)